src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -74,7 +108,12 @@ public LocalDate addWeeks(LocalDate localDate, Integer numberOfWeeks) {
      * - "someDate is today" - если someDate - сегодня
      */
     public String beforeOrAfter(LocalDate someDate) {
-        return someDate + "is today";
+        LocalDate todayDate = LocalDate.now();
+        if (someDate.isEqual(todayDate)) {
+            return todayDate + " is today";
+        }
+        return someDate.isBefore(todayDate) ? someDate + " is before " + todayDate
+                : someDate + " is after " + todayDate;
```

Comment:
let's write a single ternary operator - it can have another condition after `:`

Reply:
done